XbSymbolDatabase.c

```diff
@@ -305,8 +306,8 @@ bool CompareOOVPAToAddress(OOVPA *Oovpa, uint32_t cur)
 
 // locate the given function, searching within lower and upper bounds
 uint32_t XbSymbolLocateFunction(OOVPA *Oovpa,
-                                uint32_t lower,
-                                uint32_t upper)
+                                size_t lower,
+                                size_t upper)
```

Comment:
Not sure if `size_t` is such a great choice:

If this is C99+ anyway, then we have stdint, which provided `uintptr_t` for an integer large enough to hold a pointer.

However, for user-facing APIs, this should be using `void*` (also available pre-C99) for host addresses; and `uint32_t` (or a `typedef` of such) for Xbox addresses.

Reply:
Hmm... I don't see uintptr_t in stdint header file from Window's end. However, I do find it in vadefs.h header file.

uintptr_t seems to be what I was originally looking for both 32 and 64 bit memory range compatible. I'll replace it soon.